organization/governance-model.md

```diff
@@ -8,7 +8,7 @@ This provides a non-legal summary of the Foundation's governance model as establ
 
 ## Overview 
 
-The Foundation for Public Code is a member-owned association (vereniging) under Dutch law.
+The Foundation for Public Code is a [member-owned association (vereniging) under Dutch law](https://business.gov.nl/starting-your-business/choosing-a-business-structure/association/).
```

Comment:
I feel that membership being open only to public organisations here would be valuable to add.

Reply:
Yup!